src/story/prologue.cpp

```diff
@@ -1,6 +1,7 @@
 #include "./prologue.h"
 #include "../utils/utils.h"
 #include "../battle/battle.h"
+#include "../battle/tutorial_battle.h"
 #include "../creature/creature.h"
```

Comment:
Same here. `Creature` is not used in this file as it stands, it can probably be removed.